recipes/stages/classification/class_incr.yaml

```diff
@@ -16,13 +16,16 @@ optimizer:
     momentum: 0.9
     weight_decay: 0.0005
 
-evaluation: 
+evaluation:
     metric: ['accuracy', 'class_accuracy']
 
-task_adapt: 
+task_adapt:
     type: 'mpa'
     op: 'REPLACE'
 
 custom_hooks:
 - type: NoBiasDecayHook
   priority: 45
+
+seed: 5
+deterministic: True
```

Comment:
Can you explain why seed freezing is needed? (including deterministic true)

Reply:
We need to freeze seeds to have a reproducible code. The results on small data vary a lot (+-10%), it is not appropriate. We can't compare and measure the performance of the model because of the big random error.

Reply:
But also we can experience -10% performance degradation for edge case in our product. I think seed freezing should be used only for experiment purpose, not for product. It will be better relying the expected value for random seed instead of fixed performance gain or degradation.

Reply:
with random seed we also will experience +- 10% in some cases. Moreover, from customer point of view it is not good when you rerun training and obtain different metric. When we have small data it could be a lot different.
@goodsong81 , what do you think about seeds freezing?